packages/recurring-payments-job/package-lock.json

```diff
@@ -47,7 +47,7 @@
     },
     "node_modules/@defra-fish/business-rules-lib": {
       "version": "1.47.0-rc.7",
-      "resolved": "https://registry.npmjs.org/@defra-fish/business-rules-lib/-/business-rules-lib-1.47.0-rc.7.tgz",
```

Comment:
one question as unsure on how will work, when this merges will it then become outdated as other packages be using 1.49.0-rc.7 ?

Reply:
I'm pretty certain that the answer is yes - we'll need to update our workflow to manually update this when business-rules-lib or connectors-lib change, or we have a new release version